web/json_options.go

```diff
@@ -20,20 +20,51 @@ import "time"
 // JSONOption values passed to ConvertJSONObjectList.
 type jsonOptions struct {
 	complexAttributeNameDelimiter string
-	dateTimeFormats               []string
+	dateTimeFormats               []DateTimeFormatWithTz
+
+	// localTimeZoneOffset is the default local timezone offset that should
+	// be used for parsing date-time attributes lacking any time zone info.
+	// This should be set to the number of seconds east of UTC. If this is
+	// set to 0, this will default to UTC.
+	localTimeZoneOffset int
+}
+
+// DateTimeFormatWithTz represents a valid date time format to try parsing
+// date-time attribute values from strings.
+type DateTimeFormatWithTz struct {
```

Comment:
Nit. I would prefer to spell out `TimeZone` to be consistent everywhere.
```suggestion
type DateTimeFormatWithTimeZone struct {
```